README.md

```diff
@@ -52,6 +52,18 @@ There are 4 key features we plan to implement in the near term.
 
 For more detail on these features and where we are going, see [our roadmap](ROADMAP.md).
 
+## supported and experimental usage modes
+
+_cellxgene_ is built on standard web technologies, but is currently designed as a single-user desktop application. 
+Some of our users have experimented with hosting _cellxgene_ either for their lab or for public use, but please note that the _cellxgene_ team does not officially support, troubleshoot, or maintain any web deployments at this time.
```

Comment:
I think you should add a sentence on why we made that choice since it can seem inconceivable for most users that we are not pursuing that mode currently.